### PR TITLE
fix: only show placeholder when adding

### PR DESCRIPTION
**Pull request by thisislawatts**
PR Checklist

- [ ] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Developer experience (improves developer workflows for contributing to the project)

## Description

_What this PR does_

## Git Issues

Closes #

## Screenshots/Videos

_If useful, provide screenshot or capture to highlight main changes_

---

## What happens next?

Thanks for the contribution! We try to make sure all PRs are reviewed ahead of a monthly dev call (first Monday of the month, open to all!).

If the PR is working as intended it'll be merged and included in the next platform release, if not changes will be requested and re-reviewed once updated.

If you need more immediate feedback you can try reaching out on Discord in the [Community Platform `development` channel](https://discord.com/channels/586676777334865928/938781727017558018).

### CODE REVIEW

File: src/pages/Research/Content/Common/Update.form.tsx [hunk 1]
```diff
@@ -343,18 +344,13 @@ const UpdateForm = observer((props: IProps) => {
...
-                        {
-                          isActive: false,
-                          title: values.title || 'Title of this update',
-                          slug: null,
-                        },
-                      ]}
+                      updates={getResearchUpdates(
+                        store.activeResearch?.updates ||
+                          store.activeResearchItem.updates ||
+                          [],
```
Comment (AlfonsoGhislieri) [inline src/pages/Research/Content/Common/Update.form.tsx:350]: **Question**: I noticed this is similar to the changes in #2206, why are we also checking for `store.activeResearch?.updates` here?

Comment (thisislawatts) [reply src/pages/Research/Content/Common/Update.form.tsx]: Good catch @AlfonsoGhislieri, can you take another look?

File: src/stores/Research/research.store.tsx [hunk 2]
```diff
@@ -33,6 +33,9 @@ import type { DocReference } from '../databaseV2/DocReference'
 const COLLECTION_NAME = 'research'
 
 export class ResearchStore extends ModuleStore {
+  /**
+   * @deprecated
+   */
```
Comment (AlfonsoGhislieri) [inline src/stores/Research/research.store.tsx:38]: **Question**: has `activeResearch` been replaced by `activeResearchItem`?

**Review by AlfonsoGhislieri — APPROVED** [review]
Looks good @thisislawatts thanks for making the change!

**Pull request merged**